.test-infra/jenkins/job_beam_PerformanceTests_FileBasedIO_IT_HDFS.groovy

```diff
@@ -0,0 +1,151 @@
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import common_job_properties
+
+def testsConfigurations = [
+        [
+                jobName           : 'beam_PerformanceTests_TextIOIT_HDFS',
+                jobDescription    : 'Runs PerfKit tests for TextIOIT on HDFS',
+                itClass           : 'org.apache.beam.sdk.io.text.TextIOIT',
+                bqTable           : 'beam_performance.textioit_hdfs_pkb_results',
+                prCommitStatusName: 'Java TextIO Performance Test on HDFS',
+                prTriggerPhase    : 'Run Java TextIO Performance Test HDFS',
+                extraPipelineArgs: [
+                        numberOfRecords: '1000000'
+                ]
+
+        ],
+        [
+                jobName            : 'beam_PerformanceTests_Compressed_TextIOIT_HDFS',
+                jobDescription     : 'Runs PerfKit tests for TextIOIT with GZIP compression on HDFS',
+                itClass            : 'org.apache.beam.sdk.io.text.TextIOIT',
+                bqTable            : 'beam_performance.compressed_textioit_hdfs_pkb_results',
+                prCommitStatusName : 'Java CompressedTextIO Performance Test on HDFS',
+                prTriggerPhase     : 'Run Java CompressedTextIO Performance Test HDFS',
+                extraPipelineArgs: [
+                        numberOfRecords: '1000000',
+                        compressionType: 'GZIP'
+                ]
+        ],
+        [
+                jobName           : 'beam_PerformanceTests_AvroIOIT_HDFS',
+                jobDescription    : 'Runs PerfKit tests for AvroIOIT on HDFS',
+                itClass           : 'org.apache.beam.sdk.io.avro.AvroIOIT',
+                bqTable           : 'beam_performance.avroioit_hdfs_pkb_results',
+                prCommitStatusName: 'Java AvroIO Performance Test on HDFS',
+                prTriggerPhase    : 'Run Java AvroIO Performance Test HDFS',
+                extraPipelineArgs: [
+                        numberOfRecords: '1000000'
+                ]
+        ],
+//        [
+//                jobName           : 'beam_PerformanceTests_TFRecordIOIT_HDFS',
+//                jobDescription    : 'Runs PerfKit tests for beam_PerformanceTests_TFRecordIOIT on HDFS',
```

Comment:
Please add a comment with a link to a JIRA explaining why this is commented out.

Reply:
OK.